TAScheduler/migrations/0001_initial.py

```diff
@@ -0,0 +1,97 @@
+                ('max_assignments', models.IntegerField(default=6, validators=[django.core.validators.MaxValueValidator(6), django.core.validators.MinValueValidator(0)])),
+            ],
+        ),
+        migrations.CreateModel(
+            name='TA',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('grader_status', models.BooleanField()),
+                ('max_assignments', models.IntegerField(default=6, validators=[django.core.validators.MaxValueValidator(6), django.core.validators.MinValueValidator(0)])),
+            ],
+        ),
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('email_address', models.CharField(max_length=90)),
+                ('password', models.CharField(max_length=30)),
+                ('first_name', models.CharField(max_length=30)),
+                ('last_name', models.CharField(max_length=30)),
+                ('home_address', models.CharField(max_length=90)),
+                ('phone_number', models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='TAToCourse',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('course', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='TAScheduler.course')),
+                ('ta', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='TAScheduler.ta')),
+            ],
+        ),
+        migrations.AddField(
+            model_name='ta',
+            name='user',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='TAScheduler.user'),
+        ),
+        migrations.CreateModel(
+            name='Section',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('section_id', models.IntegerField()),
+                ('location', models.CharField(max_length=30)),
+                ('meeting_time', models.DateTimeField()),
+                ('credits', models.IntegerField()),
+                ('course', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='TAScheduler.course')),
+                ('instructor', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='TAScheduler.instructor', unique=True)),
+                ('ta', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='TAScheduler.ta', unique=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='InstructorToCourse',
```

Comment:
Rock said that an instructor can have more than 1 instructor ... however, I assumed that this is a 1 to many relationship (an instructor can have many courses but a course can only have 1 instructor). 

I am honestly not sure what to do with this - PLEASE LET ME KNOW WHAT YOU DECIDE TO DO ... it could change the prototypes/states

Reply:
We can have two instructors in a course both teaching a different section therefore we have a many to many relationship